changelog/7346.feature.rst

```diff
@@ -0,0 +1 @@
+Version information as defined by `PEP 440 <https://www.python.org/dev/peps/pep-0440/#version-specifiers>`_ may now be included when providing plugins to the ``required_plugins`` configuration option.
```

Comment:
Possibly we should include this in the changelog which added `required_plugins`, since it's the same release? (I'm OK either way)

Reply:
No strong feelings for either as well, but I think splitting would document the multiple github issues that were used for the `required_plugins` feature better ( since we arrange them one issue per line item ). Otherwise, we don't have easy to follow documentation for what closed the linked issue